source/GameInterface/Services/LoadingScreen/Messages/ShowLoadingScreen.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Common.Messaging;
+using GameInterface.Services.LoadingScreen.Handlers;
+
+namespace GameInterface.Services.LoadingScreen.Messages;
+
+/// <summary>
+/// ShowLoadingScreen to publish with broker
+/// </summary>
+public class ShowLoadingScreen : ICommand
+{
+    //The game's loading screen controls are too unstable, this is the only way I got it to work after trying many different approaches.
+    public ShowLoadingScreen()
+    {
+        var loadingScreenHandler = new CoopLoadingScreenHandler(MessageBroker.Instance);
+    }
+}
```

Comment:
The handler is automatically created [here](https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/development/source/GameInterface/Services/ServiceModule.cs#L19)

```suggestion
public record ShowLoadingScreen : ICommand
{
}
```